Steer/DigitizerWorkflow/src/ITSMFTDigitizerSpec.cxx

```diff
@@ -189,8 +191,21 @@ class ITSMFTDPLDigitizerTask : BaseDPLDigitizer
       return;
     }
     if (matcher == ConcreteDataMatcher(mOrigin, "DEADMAP", 0)) {
-      LOG(info) << mID.getName() << " dead map updated";
-      mDigitizer.setDeadChannelsMap((const o2::itsmft::NoiseMap*)obj);
+      LOG(info) << mID.getName() << " static dead map updated";
+      mDeadMap = (o2::itsmft::NoiseMap*)obj;
+      mDigitizer.setDeadChannelsMap(mDeadMap);
+      return;
+    }
+    if (matcher == ConcreteDataMatcher(mOrigin, "TimeDeadMap", 0)) {
+      o2::itsmft::TimeDeadMap* timedeadmap = (o2::itsmft::TimeDeadMap*)obj;
+      if (!timedeadmap->isDefault()) {
+        timedeadmap->decodeMap(mFirstOrbitTF, mDeadMap, true);
```

Comment:
just for the record: the update of the mDeadMap via decodeMap can be done only once (to avoid an accumulation of maps from different timestamps). This is ok in this case since the DigitizerSpec processes only 1 TF. 
To avoid surprises if one day this changes, one can probably add here a check:
```
static bool UpdateDone = false;
if (UpdateDone) {
  LOGP(fatal, "Attempt to add time-dependent map to already modified static map");
}
UpdateDone = true;
```

Reply:
Hi @shahor02 , I'm not completely sure how the loop works. Should this flag UpdateDone be a data member of the class.. or only defined in the `scope of finaliseCCDB` ?

Reply:
No, I meant a local static variable, right after the L202